src/components/ChatWindow.js

```diff
@@ -14,8 +14,8 @@ class ChatWindow extends Component {
       this.props.onUserInputSubmit(message);
     }
 
-    onMessageReceived(message) {
-      this.setState({ messages: [...this.state.messages, message] });
+    onFilesSelected(filesList) {
+      this.props.onFilesSelected(filesList);
```

Comment:
What's up with this why isn't it in the PropTypes? (seems like that's the case for multiple props)

Reply:
should be.. let's put it there.